bots/images/scripts/debian.setup

```diff
@@ -111,7 +111,7 @@ eatmydata apt-get -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
 
 # extract control files and adjust them for our release, so that we can parse the build deps
 mkdir -p /tmp/out
-tar -C /tmp/out --strip-components=1 --wildcards -xf "$TEST_SOURCE" '*/debian/'
+curl -L https://github.com/cockpit-project/cockpit/archive/master.tar.gz | tar -C /tmp/out --strip-components=1 --wildcards -zxf - '*/debian/'
```

Comment:
Note to self: Originally I thought `curl https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/debian/control`, but adjust-for-release removes a build dependency. So this is fine (and not that much more overhead).